core/src/main/scala/org/apache/spark/ui/jobs/AllJobsPage.scala

```diff
@@ -465,8 +465,7 @@ private[ui] class JobDataSource(
     val jobDescription =
       UIUtils.makeDescription(
         jobData.description.getOrElse(lastStageDescription),
-        basePath,
-        plainText = false)
```

Comment:
There are many similar cases in the code, and I don't think it's worth cleaning up specifically. Even if it's really necessary, it shouldn't be just cleaning up this one

Reply:
+1.
I don't think it's worth too.